server/controllers/admin/suppliers.js

```diff
@@ -66,26 +78,50 @@ function detail(req, res, next) {
   .done();
 }
 
-// POST /supplier
+/**
+ * @method create
+ *
+ * @description
+ * POST /supplier
+ *
+ * This method creates a new supplier entity in the database and sets up the
+ * creditor for the it.
+ */
 function create(req, res, next) {
   const data = db.convert(req.body, ['creditor_uuid']);
 
   // provide uuid if the client has not specified
-  data.uuid = db.bid(data.uuid || uuid());
+  const uid = data.uuid || uuid();
```

Comment:
here the `uuid` library is directly invoked with `uuid()` - in another file (`location.js` I think) `uuid.v4()` is used to retrieve a uuid. Should we stick to one of these methods?